R/bayesian/result_details_panel.R

```diff
@@ -6,60 +6,30 @@
 result_details_panel_ui <- function(id) {
   ns <- NS(id)
   div(
-    helpText(
-      "Please note: if you change the selections on the sidebar,
-      you will need to re-run the primary and/or sensitivity analysis from the 'Forest Plot' page."
-    ),
-    fluidRow(
-      column(
-        width = 6,
-        p(tags$strong("Results details for all studies")),
-        verbatimTextOutput(outputId = ns("gemtc_results")),
-        p(tags$strong("Gelman convergence assessment plot for all studies")),
-        plotOutput(outputId = ns("gemtc_gelman"))
-      ),
-      column(
-        width = 6,
-        p(tags$strong("Results details with studies excluded")),
-        verbatimTextOutput(outputId = ns("gemtc_results_sub")),
-        p(tags$strong("Gelman convergence assessment plot with studies excluded")),
-        plotOutput(outputId = ns("gemtc_gelman_sub"))
-      )
-    )
+    p(tags$strong("Results details for all studies")),
+    verbatimTextOutput(outputId = ns("gemtc_results")),
+    p(tags$strong("Gelman convergence assessment plot for all studies")),
```

Comment:
The two titles here need to vary depending on whether its a sensitivity analysis or not (currently the main Bayesian tab says 'for all studies' twice)

Reply:
Changed to follow the same pattern as other tabs.